<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AlarmQueryForm</class>
 <widget class="QWidget" name="AlarmQueryForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>480</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>55</y>
      <width>701</width>
      <height>350</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>WenQuanYi Micro Hei</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::WheelFocus</enum>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="iconSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>历史故障</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>700</width>
        <height>320</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>WenQuanYi Micro Hei</family>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <column>
       <property name="text">
        <string>序号</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>故障名称</string>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>开始时间</string>
       </property>
       <property name="font">
        <font>
         <family>WenQuanYi Micro Hei</family>
         <pointsize>12</pointsize>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>结束时间</string>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>历史告警</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget_1">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>700</width>
        <height>320</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>WenQuanYi Micro Hei</family>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <column>
       <property name="text">
        <string>序号</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>告警名称</string>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>开始时间</string>
       </property>
       <property name="font">
        <font>
         <family>WenQuanYi Micro Hei</family>
         <pointsize>12</pointsize>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>结束时间</string>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>工作状态记录</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>700</width>
        <height>321</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>WenQuanYi Micro Hei</family>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <column>
       <property name="text">
        <string>序号</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>状态信息</string>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>开始时间</string>
       </property>
       <property name="font">
        <font>
         <family>WenQuanYi Micro Hei</family>
         <pointsize>12</pointsize>
        </font>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>操作记录</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>701</width>
        <height>321</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>序号</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>操作信息</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>时间</string>
       </property>
      </column>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="historypushButton">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>410</y>
      <width>50</width>
      <height>51</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>WenQuanYi Micro Hei</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_endpage">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>410</y>
      <width>85</width>
      <height>38</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>WenQuanYi Micro Hei</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>末页</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_homepage">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>410</y>
      <width>85</width>
      <height>38</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>WenQuanYi Micro Hei</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>首页</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_uppage">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>410</y>
      <width>85</width>
      <height>38</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>WenQuanYi Micro Hei</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>上一页</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_nextpage">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>410</y>
      <width>85</width>
      <height>38</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>WenQuanYi Micro Hei</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>下一页</string>
    </property>
   </widget>
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>275</x>
      <y>0</y>
      <width>250</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>历史信息</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>pushButton_homepage</tabstop>
  <tabstop>pushButton_uppage</tabstop>
  <tabstop>pushButton_nextpage</tabstop>
  <tabstop>pushButton_endpage</tabstop>
  <tabstop>historypushButton</tabstop>
  <tabstop>tableWidget_3</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
